lib/modules/dosomething/dosomething_signup/dosomething_signup.module

```diff
@@ -13,12 +13,17 @@ include_once 'dosomething_signup.theme.inc';
 include_once 'includes/dosomething_signup.mobilecommons.inc';
 include_once 'includes/dosomething_signup.variable.inc';
 include_once 'includes/dosomething_signup.cron.inc';
+include_once 'dosomething_signup.query.inc';
+
+include_once 'includes/Signup.php';
+include_once 'includes/SignupController.php';
```

Comment:
Reference to **SignupController.php** which doesn't exist in the **/includes** directory is throwing errors!